Task2/Task2/Task2/Infrastructure/JsonWriter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Newtonsoft.Json;
+using Task2.Products;
+
+namespace Task2.Infrastructure
+{
+    public static class JsonWriter<T>
+    {
+        public static void JsonWriterInFile(List<T> listBricks, string path)
```

Comment:
The same situation as in JsonReader.
And `listBricks` on `item`.

Reply:
fixed